test/unit/org/apache/cassandra/db/ColumnFamilyStoreTest.java

```diff
@@ -44,6 +44,10 @@
 import org.apache.cassandra.SchemaLoader;
 import org.apache.cassandra.UpdateBuilder;
 import org.apache.cassandra.Util;
+
```

Comment:
changes in this class happen to be not related to this ticket at all, revert these changes on commit